lib/assert.js

```diff
@@ -31,8 +33,21 @@ exports.exists = (value, message) => {
 };
 
 exports.isPositiveNumber = (value, message) => {
```

Comment:
This is a bit odd in that the default return value is undefined which gets returned if the number is positive. It passes the assert because assert checks for 0 or false (not undefined). Mostly a style thing but if it were me I'd try to make the code as clear as possible and return true if the number is positive. Also, returning undefined prevents someone from testing for true when calling this method (standard coding style).